<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Foreingzone</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 85, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QFrame" name="loginFrame">
          <property name="styleSheet">
           <string notr="true">

#loginFrame{

border: 3px solid black;
border-radius: 40px;
background-color:rgb(255,255,255);


}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="loginLabel">
           <property name="geometry">
            <rect>
             <x>12</x>
             <y>12</y>
             <width>142</width>
             <height>25</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 85, 0);
background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Iniciar Sesión</string>
           </property>
          </widget>
          <widget class="QLabel" name="emailLabel">
           <property name="geometry">
            <rect>
             <x>12</x>
             <y>111</y>
             <width>69</width>
             <height>25</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Correo </string>
           </property>
          </widget>
          <widget class="QLineEdit" name="emailLE">
           <property name="geometry">
            <rect>
             <x>12</x>
             <y>142</y>
             <width>201</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 85, 0);</string>
           </property>
           <property name="placeholderText">
            <string>alias@hotmail.com</string>
           </property>
          </widget>
          <widget class="QLabel" name="passwordLabel">
           <property name="geometry">
            <rect>
             <x>12</x>
             <y>179</y>
             <width>88</width>
             <height>25</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="passwordLE">
           <property name="geometry">
            <rect>
             <x>12</x>
             <y>210</y>
             <width>201</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>Contraseña</string>
           </property>
          </widget>
          <widget class="QPushButton" name="loginPB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>12</x>
             <y>247</y>
             <width>134</width>
             <height>33</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 170, 0);</string>
           </property>
           <property name="text">
            <string>Iniciar Sesión</string>
           </property>
          </widget>
          <widget class="QLabel" name="newUserLabel">
           <property name="geometry">
            <rect>
             <x>12</x>
             <y>332</y>
             <width>88</width>
             <height>19</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 255);
background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Eres Nuevo?</string>
           </property>
          </widget>
          <widget class="QPushButton" name="newUserPB">
           <property name="geometry">
            <rect>
             <x>12</x>
             <y>357</y>
             <width>149</width>
             <height>27</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Crear Cuenta Nueva</string>
           </property>
          </widget>
          <widget class="QLabel" name="ErrorDeCorreo">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>290</y>
             <width>111</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QFrame" name="newUserFrame">
          <property name="styleSheet">
           <string notr="true">

#newUserFrame {
border: 3px solid black;
border-radius: 40px;
	background-color: rgb(255, 255, 255);
}
 
</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="createUserLabel">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 85, 0);
background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Crear Cuenta</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="newNamLB">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Nombre</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="newNameLE">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="placeholderText">
              <string>Juan Perez</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="newMailLabel">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Correo</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLineEdit" name="newMailLE">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="placeholderText">
              <string>alias@gmail.com</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="newPasswordLabel">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Contraseña</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLineEdit" name="newPasswordLE">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="newPasswordLabelC">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Verificacion de contraseña</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLineEdit" name="newPasswordLEC">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="newError">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QPushButton" name="createPB">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">#createPB{
background-color: rgb(255, 170, 0);
}</string>
             </property>
             <property name="text">
              <string>Crear Cuenta</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QComboBox" name="OrdenBox">
          <item>
           <property name="text">
            <string>-</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Descendente</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ascendente</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEditBuscar"/>
        </item>
        <item row="2" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>661</width>
             <height>195</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_5"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QComboBox" name="TipoBox">
          <item>
           <property name="text">
            <string>Todos los departamentos</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Comida</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Deportes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Electronico</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hogar</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Libros</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QFrame" name="frameNuevo">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>700</width>
            <height>400</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollDestacados">
             <property name="minimumSize">
              <size>
               <width>10</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>700</width>
               <height>400</height>
              </size>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>641</width>
                <height>175</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_8">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayoutNuevo"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Logo">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10777215</width>
        <height>10777215</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
